src/editor/ui_menus/main_menu_random_map.cc

```diff
@@ -556,10 +556,21 @@ bool MainMenuNewRandomMap::do_generate_map(Widelands::EditorGameBase& egbase,
 
 	Widelands::Map* map = egbase.mutable_map();
 
+	// TODO(Nordfriese): Perhaps support add-ons in the future? For now we just disable them all
```

Comment:
We have no idea how random maps and addons should wor togehter so better just disable them all